cmd/edenConfig.go

```diff
@@ -55,6 +56,19 @@ func reloadConfigDetails() {
 		qemuFileToSave = utils.ResolveAbsPath(viper.GetString("eve.qemu-config"))
 		devModel = viper.GetString("eve.devmodel")
 		eveRemote = viper.GetBool("eve.remote")
+		modelFile = viper.GetString("eve.devmodelfile")
+		if modelFile != "" {
+			filePath, err := filepath.Abs(modelFile)
+			if err != nil {
+				log.Errorf("cannot get absolute path for devmodelfile (%s): %v", modelFile, err)
+				filePath = ""
+			} else
+			if _, err := os.Stat(filePath); err != nil {
```

Comment:
Doesn't yetus complain about this style? "else if" on a single line is what I'd expect.